data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	The contestants are recalled back to their rafts, where they are provided with water and given time to rest. During their break, three rafts carrying cargo land, while those holding the nets in place below the rope web rearrange themselves to form a long, straight line. The cargo rafts are unloaded to reveal bulky, and clearly heavy, suits of armor that the remaining contestants don after their break, while those disqualified leave on the the cargo rafts. Two larger rafts with a thin, metallic pipe connecting them then approach and set themselves in position over the line of nets. The Arachi spread across the thin pipe, and the helpful Saryd explains that they must perform the difficult task of maintaining their balance on the narrow surface with their weighty armor as long as possible.` 
+			`	"A test of one's endurance, this stage is," the Saryd says. "Proud of both their ships' and their own resilience, the Arachi are. To hold on the longest, the objective is." Most of the Arachi are able to stay on for roughly a minute, but soon after that the competitors begin falling one after another. At the two minute mark, only four are left, but all fall only a few seconds later.`
+			`	Once the results from the two contests are merged, Tumub has climbed to eighth. He and the other Arachi in the top twenty all celebrate enthusiastically, but as dark storm clouds gather on the horizon and rapidly begin approaching the cluster of trees, it isn't long before everyone is back on rafts returning to the port. When you step off the raft, you see that aside from the train that brought you here, another one is now ready to depart. As the Saryd who sat beside you leaves also disembarks, she explains that it leads to the grounds where the cooperative game will happen, but that there should still be time for you to return and see the other games.`
+				goto choice
+			
+			
+			label wp
+			`	Unlike the other trains, this one blatantly mimics Heliarch warships in design, clearly meshing the three styles of Coalition ships. It brings you to a large hangar, where you're instructed to board one of hundreds of ring-shaped flying rafts available. All seats face the inside of the ring, and you take the one that looks best suited for a human. As you idly watch the mechanics here perform various preflight safety checks on the raft, you hear what you by now recognize as Kimek speech from the seat to your right.`
```

Comment:
What I suppose you could call a running joke is that all three Coalition species have very different bodies, and as such "chairs" for them would be different enough that it'd be annoying to use the chair of another species, so they have adjustable seats that can be rearranged to better fit any of the three.
Humans have different bodies than all three species, though, and as such it would be annoying to use seats in any of the three configurations, so you need to fiddle with the seats to have them comfortable. The "best suited for a human" part is saying one of the seats was left in a configuration that's easier to change into a regular human seat.
It's just another way of saying the same "the parts of these seats can be changed and you change them to be usable by humans bla bla" bit, without repeating it again.

Reply:
Hmm, the writing here seems to imply that there was a seat there custom-built for a human. Perhaps you could change the wording to suggest that a seat was already configured in such a way that first last doesn't have to do much adjusting to make it fit them.

Reply:
Is what I changed it to better?

Reply:
Yes, much better.